src/components/unmute-video-button.js

```diff
@@ -5,9 +5,19 @@ AFRAME.registerComponent("unmute-video-button", {
     this.onClick = () => {
       const videoEl = findAncestorWithComponent(this.el, "media-video");
       const mediaVideo = videoEl.components["media-video"];
-      if (!mediaVideo || !mediaVideo.video) return;
-      mediaVideo.video.muted = false;
-      this.el.setAttribute("visible", false);
+      if (!mediaVideo || !mediaVideo.video || !mediaVideo.videoMutedAt) return;
+
+      // iOS initially plays the sound and *then* mutes it, and sometimes a second video playing
+      // can break all sound in the app. (Likely a Safari bug.) Adding a delay before the unmute
+      // occurs seems to help with reducing this.
+      if (performance.now() - mediaVideo.videoMutedAt < 3000) {
```

Comment:
I might normally complain about this sort of thing, but having done iOS web audio debugging in a past life, this is a very tame hack, so.. nothing to see here.. lol